x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/helpers.tsx

```diff
@@ -342,3 +342,18 @@ export const addContentToTimeline = ({
     });
   }
 };
+
+export const getDisplayValue = (
+  value: string | number | Array<string | number>
+): string | number => {
+  if (isStringOrNumberArray(value)) {
+    return `( ${value.join(' OR ')} )`;
+  }
+  return value;
+};
+
+function isStringOrNumberArray(
```

Comment:
consider applying the feedback [in this comment](https://github.com/elastic/kibana/pull/142436#discussion_r987095667) here